services/connectivity/messaging/src/main/java/org/eclipse/ditto/services/connectivity/messaging/amqp/StatusReport.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.services.connectivity.messaging.amqp;
+
+import java.util.Optional;
+import javax.annotation.Nullable;
+import javax.jms.MessageConsumer;
+import javax.jms.MessageProducer;
+import org.eclipse.ditto.services.connectivity.messaging.internal.ConnectionFailure;
+
+/**
+ * Message for status reporter.
+ */
+public class StatusReport {
```

Comment:
could you please make this class package private and `final`?

Reply:
done